.github/workflows/tests.yml

```diff
@@ -0,0 +1,76 @@
+name: Tests
+
+on:
+    push: ~
+    pull_request: ~
+
+jobs:
+    phpunit:
+        name: PHPUnit on ${{ matrix.php }}
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['7.1', '7.2', '7.3', '7.4']
+                coverage: [true]
+                composer-flags: ['']
+                include:
+                    - php: '8.0'
+                      coverage: false
+                      composer-flags: '--ignore-platform-reqs'
```

Comment:
Better to use `--ignore-platform-req=php` (requires composer 2).

Reply:
Thank you for the tip!